src/rebar_prv_common_test.erl

```diff
@@ -480,52 +517,81 @@ copy_bare_suites(From, To) ->
     ok = rebar_file_utils:cp_r(SrcFiles, To),
     rebar_file_utils:cp_r(DataDirs, To).
 
+maybe_copy_spec(State, [App|Apps], Spec) ->
+    case rebar_file_utils:path_from_ancestor(filename:dirname(Spec), rebar_app_info:dir(App)) of
+        {ok, []}   ->
+            ok = rebar_file_utils:cp_r([Spec],rebar_app_info:out_dir(App));
+        {ok,_} ->
+            ok;
```

Comment:
I think this should be covered by https://github.com/sirihansen/rebar3/blob/4e52ce58e505ba57f2188e34a002d70de1bd6ff5/src/rebar_prv_common_test.erl#L437, i.e. all directories where specs are found are added as `extra_src_dirs`, except if it is the project- or app root dir (which is filtered out by `maybe_inject_test_dir/4`). I first did an attempt to copy specs along with the suites in `copy_bare_suites/2`, but since there is no naming rule for specs it didn't make sense - that's why I added `maybe_copy_spec/3`.